miden-lib/asm/sat/constants.masm

```diff
@@ -13,6 +13,9 @@ const.NOTE_MEM_SIZE=1024
 # The depth of the Merkle tree used to commit to notes produced in a block.
 const.NOTE_TREE_DEPTH=20
 
+# The maximum number of notes that can be created in a single transaction (2^NOTE_TREE_DEPTH)
+const.MAX_NUM_CREATED_NOTES=1048576
```

Comment:
I'm thinking this should be a smaller number - maybe something like $2^{10}$ or $2^{12}$.

Reply:
Ok happy to change in combination with the `NOTE_TREE_DEPTH`.  Do you have a preference over $2^10$ or $2^12$?

Reply:
Makes sense, thanks for the explanation.  I will update accordingly.

Reply:
done